src/API/Management/Blacklists.php

```diff
@@ -15,7 +15,11 @@ public function getAll($aud)
     {
         $response = $this->httpClient->get('/blacklists/tokens?'.http_build_query(['aud' => $aud]));
 
-        return ResponseMediator::getContent($response);
+        if (200 === $response->getStatusCode()) {
```

Comment:
Do we really want to put that logic in the ResponseMediator? That violates single responsibility principle. The ResponseMediator only converts PSR7 Responses with content type JSON to an array. I do not think it should handle exceptions. That is domain logic.